### PR TITLE
Upgrade to React-Flow v11

### PR DESCRIPTION
**Pull request by joeyballentine**
This gives us a few things
- A cool grabbing cursor when panning
- The ability to middle click drag to pan, even on nodes
- Much better performance when dragging around nodes, for some reason

Problems I've found so far:
- ~Panning has much worse performance now, for some reason. Going to guess it has something to do with a viewport related hook or something.~
- Occasionally the grab cursor won't change while dragging, but changes after

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -217,10 +217,11 @@
     "re-resizable": "^6.9.9",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
-    "react-flow-renderer": "^10.3.16",
+    "react-flow": "^1.0.3",
```
Comment (RunDevelopment) [inline package.json:220]: [Uhmm](https://www.npmjs.com/package/react-flow)

File: src/renderer/global.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 @use "colors.scss";
+@import "reactflow/dist/style.css";
```
Comment (RunDevelopment) [inline src/renderer/global.scss:2]: Does this even work? Can you use npm-style imports in SCSS files?

File: src/renderer/global.scss [hunk 3]
```diff
@@ -129,12 +130,12 @@ body {
     width: 8px;
 }
 
-[data-theme='dark'] ::-webkit-scrollbar-thumb {
+[data-theme="dark"] ::-webkit-scrollbar-thumb {
```
Comment (RunDevelopment) [inline src/renderer/global.scss:133]: I'm pretty sure we use single-quotes everywhere, so why? Did your default formatter do that? If so, then maybe we should add stylelint. Since we plan on moving away from CSS-in-JS, adding stylelint might be a good move regardless.

File: package.json (hunk 1 above)
Comment (joeyballentine) [reply package.json:220]: Whoops

File: src/renderer/global.scss (hunk 2 above)
Comment (joeyballentine) [reply src/renderer/global.scss:2]: Yes. Not only does it work, it's necessary.

File: src/renderer/global.scss (hunk 3 above)
Comment (joeyballentine) [reply src/renderer/global.scss:133]: Different pr please, but yes. I didn't do this on purpose so it has to have been formatter

**Comment by joeyballentine** [comment]
I think we should just merge this. It doesn't seem to break anything and having to keep fixing merge conflicts on it is getting annoying. Plus, we probably won't release anything for a while considering the tile size issues.

File: src/renderer/components/ReactFlowBox.tsx [hunk 4]
```diff
@@ -425,7 +425,7 @@ export const ReactFlowBox = memo(({ wrapperRef, nodeTypes, edgeTypes }: ReactFlo
             >
                 <Background
                     gap={16}
-                    size={0.5}
+                    size={1}
```
Comment (RunDevelopment) [inline src/renderer/components/ReactFlowBox.tsx:428]: Why was this changed?

Comment (joeyballentine) [reply src/renderer/components/ReactFlowBox.tsx:428]: Because they changed something internally that made this necessary to get the old size.

**Review by RunDevelopment — APPROVED** [review]
